docs/algob-config.md

```diff
@@ -47,6 +47,15 @@ Each network configuration requires a list of accounts. These accounts are then
           mnemonic: "call boy rubber fashion arch day capable one sweet skate outside purse six early learn tuition eagle love breeze pizza loud today popular able divide"
         }]);
 
+1. Loaded from env variable `ALGOB_ACCOUNTS`
+
+        const accounts = loadFromEnv();
+        /**
+        * keys can be stored in `ALGOB_ACCOUNTS` env variable.
+        *  the keys in env variable should be JSON string of the following structure: 
+        * [{"name": "account_name", "menmonic": "mnemonic string"]} 
+        */
```

Comment:
This is a doc, so please move the comment to the normal text in the same point (after new line).

Reply:
ok